custom-login/src/Login.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import * as OktaSignIn from '@okta/okta-signin-widget';
+import { Container } from 'semantic-ui-react';
+import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css';
+import '@okta/okta-signin-widget/dist/css/okta-theme.css';
+
+import Navbar from './Navbar';
+import config from './.samples.config';
+
+export default class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.signIn = new OktaSignIn({
+      baseUrl: config.oktaSample.widget.baseUrl,
```

Comment:
nit: I'm a fan of capturing the `baseUrl` by parsing the `issuer`
>  config.oktaSample.oidc.issuer.split('/oauth2')[0]

Reply:
As noted above I prefer not to obfuscate that needed config for the widget.